lib/phoenix_live_view/html_engine.ex

```diff
@@ -648,6 +650,7 @@ defmodule Phoenix.LiveView.HTMLEngine do
     {let, [quoted_value | r], a}
   end
 
+  # TODO: deprecate "let" in favor of `:let`
```

Comment:
@josevalim are we going to deprecate the old one?

Reply:
Eventually, yes, but not now. Let's have the formatter rewrite it for a couple releases :)

Reply:
```suggestion
  # TODO: deprecate "let" in favor of `:let` on LV v0.19
```